webclient/bower_components

```diff
@@ -0,0 +1 @@
+../bower_components/
```

Comment:
Oh. I thought you were implying that this pull request would prevent us from running source, which I understood as already not possible. 

I don't see why we shouldn't use a symlink if it's helpful. The reason I added this symlink back is because there are some bower_component files that I needed to copy unchanged. If it wasn't symlinked, I would have had to add gulp code to juggle weird filepath changes for imports. I'm talking specifically about the `dontVulcanizeTheseFiles` array.